### PR TITLE
SoarETX: use bundled LibGUI

### PR DESCRIPTION
**Pull request by offer-shmuely**
SoarETX: 
Fixed running widgets due to libgui changes, now it use internal libgui to enable back the widgets
allow to run on users TX that do not have libgui installed

### CODE REVIEW

**Comment by pfeerick** [comment]
You need to sync your master branch properly at some point... you shouldn't have all the merge branch commits, nor should it be 28 commits ahead of main ;) I don't know what your workflow is, but it is straightforward enough to do in Github natively... 

image
image

That will then bring your master branch back into sync with this repo, and there won't be any extra noise in it or in any branches off of you do.

**Comment by offer-shmuely** [comment]
@pfeerick I will love to learn, it annoying me as well.


my flow is:
1. go to my github.com/offer-shmuely/edgetx-sdcard and delete my old branches
2. sync the fork
3. on my computer, on existing folder that is a clone of the /offer-shmuely/edgetx-sdcard repo, change back to master (checkout)
4. git pull
5. create a new branch
6. transfer by beyond-compare the files I like to update from my environment
7. git comit
8. git push
9. on the github.com/offer-shmuely/edgetx-sdcard press the suggest button to create PR

note that even that I sync the fork, I still see commits ahead

image

and when I like to create a PR
it look like this:
https://github.com/EdgeTX/edgetx-sdcard/compare/master...offer-shmuely:edgetx-sdcard:shmuely/model_locator_display_ver?expand=1

image


what am I missing?

**Comment by pfeerick** [comment]
Hm... the merge commits are probably confusing githubs sync into thinking it's actually in sync. 

You could possibly do the following on your computer on the master branch, to rewind it back 29 commits

`git reset --hard HEAD~29`

... then force push it ...

`git push --force`

... and then sync fork button should work properly. If it doesn't, maybe try `HEAD~30` instead (i.e. one more commit back). 

TLDR ... The "Sync fork" button is basically doing the following (upstream remote is already configured)

```
git fetch upstream
git checkout master
git merge upstream/master
```
More on that at https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork
